endpoints/openrtb2/auction.go

```diff
@@ -249,10 +257,33 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 		labels, ao = rejectAuctionRequest(*rejectErr, w, deps.hookExecutor, req.BidRequest, account, labels, ao)
 		return
 	}
-
+	setSeatNonBidRaw(req, auctionResponse)
 	labels, ao = sendAuctionResponse(w, deps.hookExecutor, response, req.BidRequest, account, labels, ao)
 }
 
+// setSeatNonBidRaw is temporary introduced function for setting SeatNonBid inside bidResponse.Ext
+//
+//	populates bidresponse.ext.prebid.seatnonbid if  bidrequest.ext.prebid.returnallbidstatus is true using setSeatNonBid()
+//
+// This function will go away in next PR of https://github.com/prebid/prebid-server/pull/2505/
+func setSeatNonBidRaw(request *openrtb_ext.RequestWrapper, aucResponse *exchange.AuctionResponse) bool {
```

Comment:
I'm confused by the use of the word `raw` in this function name? Could you explain this?

Reply:
https://github.com/prebid/prebid-server/pull/2505/files#discussion_r1150031661 
Added `Raw` as suffix as there is already function with name `setSeatNonBid`